### ISSUE TITLE
Mangle application interactions

### ISSUE THREAD

**Issue by Felynda** [SunstriderEmu/BugTracker#1931]
**Describe the bug**
On retail TBC it was possible to get a Bleed cast that would be affected by Mangle twice, see Elitist Jerks posts: https://web.archive.org/web/20081226020058/http://elitistjerks.com:80/805904-post4398.html
https://web.archive.org/web/20081109090249/http://elitistjerks.com/808149-post4418.html

Basically due to interactions between a Mangle being already up when a bleed is being cast gave it the bonus damage of this mangle as well as a mangle cast shortly afterwards.

I was not able to replicate this at all on the PTR., because this has to do with spell batching sorta like Seal Twisting. 
In short: A bleed cast within the last 0,5ish seconds of mangle debuff should be affected by both this debuff and any mangle cast after that.
